src/test/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkTest.java

```diff
@@ -304,6 +314,20 @@ public void testBigQuerySinkConfigInvalidChunkSize() {
     Assert.assertEquals(1, collector.getValidationFailures().size());
   }
 
+  @Test
+  public void testBigQuerySinkConfigNullSchemaTableNotExist() throws NoSuchFieldException {
+    BigQuerySinkConfig config = new BigQuerySinkConfig("44", "ds", "tb", "bucket", null,
+                                                       "INTEGER", 0L, 100L, 10L, "2097152");
+    FieldSetter.setField(config, GCPConfig.class.getDeclaredField("project"), "value");
+    FieldSetter.setField(config, GCPConfig.class.getDeclaredField("serviceFilePath"), "filepath");
+
+    MockFailureCollector collector = new MockFailureCollector("bqsink");
+    mockStatic(BigQueryUtil.class);
+    PowerMockito.when(BigQueryUtil.getBigQueryTable(any(), any(), any(), any(), any(), any())).thenReturn(null);
+    config.validateNullInputSchemaAndTableNotExist(null, collector);
+    Assert.assertEquals(1, collector.getValidationFailures().size());
```

Comment:
Please try to catch also the message and the attributes you are expecting to fail.

Reply:
PR updated.